py3status/modules/xscreensaver.py

```diff
@@ -11,45 +11,44 @@
 Settings can be managed using "xscreensaver-demo".
 
 Configuration parameters:
-    cache_timeout: refresh interval for this module (default 60)
     format: default format for this module (default '{output}')
-    string_off: display when Xscreensaver is not running (defaults '🖥️')
-    string_on: display when Xscreensaver is running (defaults '🖥️')
+    format_off: display when Xscreensaver is not running (default '')
+    format_on: display when Xscreensaver is running (default '')
```

Comment:
It was done immediately / silently. I didn't see what you said about dependencies until several hours later. `SCR`? Will look for square box.